app/javascript/dashboard/i18n/locale/en/report.json

```diff
@@ -411,5 +414,14 @@
       "BUSY": "Busy",
       "OFFLINE": "Offline"
     }
+  },
+  "DAYS_OF_WEEK": {
+    "SUNDAY": "Sunday",
+    "MONDAY": "Monday",
+    "TUESDAY": "Tuesday",
+    "WEDNESDAY": "Wednesday",
+    "THURSDAY": "Thursday",
+    "FRIDAY": "Friday",
+    "SATURDAY": "Saturday"
```

Comment:
@pranavrajs, there's [another place](https://github.com/chatwoot/chatwoot/blob/c998c84bc090b75f282a7b0a95b11c158e65a614/app/javascript/dashboard/routes/dashboard/settings/inbox/components/WeeklyAvailability.vue#L97-L105) where we have days of the week. But it's not translated there. What's the best place to add these?

Reply:
Can we have a common place in i18n for these kinds of items? I'm not sure about the best practices. Can you please do some research and see how we can forward?

Reply:
At the moment, since all the JSON files are imported as is and the global translation directory is flattened. It's okay to store it in the `reports.json` itself. Since all the translations are loaded in their entirety, it's fine.

An alternative is to add a `global.json` file and add it there.